apps/autonolas-registry/common-util/List/ListTable/helpers.jsx

```diff
@@ -145,9 +146,9 @@ export const getTableColumns = (
       dataIndex: 'owner',
       key: 'owner',
       width: 200,
-      render: (text) => {
+      render: (text, record) => {
         if (!text || text === NA) return NA;
-        return <AddressLink {...addressLinkProps} text={text} chainName={chainName} />;
+        return <AddressLinkSimple address={record.owner} chainId={chainId} />;
```

Comment:
the `AddressLink` from @autonolas/frontend-library creates the url based on the provided chainId, there's no way to pass a custom URL from here. Just simply replaced it with the mono repo's component, so we can see correct scan URL for mode.